components/threading/global_threads/thread_list/thread_list.tsx

```diff
@@ -82,6 +82,12 @@ const ThreadList = ({
             return;
         }
 
+        const noModalsAreOpen = document.getElementsByClassName(A11yClassNames.MODAL).length === 0;
+        const noPopupsDropdownsAreOpen = document.getElementsByClassName(A11yClassNames.POPUP).length === 0;
+        if (!noModalsAreOpen || !noPopupsDropdownsAreOpen) {
```

Comment:
What's this used for?

Reply:
Currently, if you have the modal open in the threads view and press up or down, the focus will move out of the modal into the threads. This is to prevent that from happening.

Reply:
Can you just add a comment above this piece stating the above?